src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -101,7 +127,23 @@ export const _Chatbox: React.FC<ChatboxProps> = ({
 
   return (
     <div className="Chatbox">
-      <div className="Chatbox__messages">{renderedMessages}</div>
+      <div
+        className="Chatbox__messages-container"
+        ref={scrollableComponentRef}
+        id={"Chatbox_scrollable_div"}
+      >
+        <InfiniteScroll
+          dataLength={messagesRenderCount}
```

Comment:
Any reason to both slice the messages and set `dataLength` here?

Reply:
That's how InfiniteScroll components works: [docs](https://github.com/ankeetmaini/react-infinite-scroll-component#props) - this property must be set to the number of rendered children.